src/applications/vaos/redux/selectors.js

```diff
@@ -35,10 +35,7 @@ export const selectFeatureToggleLoading = state => toggleValues(state).loading;
 // Use flat facility page for non Cerner patients
 export const selectUseFlatFacilityPage = state => !selectIsCernerPatient(state);
 
-const selectFeatureProviderSelection = state =>
-  toggleValues(state).vaOnlineSchedulingProviderSelection;
-export const selectUseProviderSelection = state =>
-  selectFeatureProviderSelection(state) &&
+export const hasVAPResidentialAddress = state =>
```

Comment:
Could we name this with a `select` prefix? I think that's helpful to distinguish the selector from the resulting boolean.